_data/simple-icons.json

```diff
@@ -10154,7 +10154,7 @@
         },
         {
             "title": "MoneyGram",
-            "hex": "FF6600",
+            "hex": "E2231A",
```

Comment:
```suggestion
            "hex": "DA291C",
```

The color should be `--themeColorRed` `#DA291C`. You can use the color pick on the logo. The `#DA291C` is the closest color.

Reply:
Good thinking, you’re right — I didn’t spend enough time comparing, just saw a reference to a “main color” and ran with that